sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -311,77 +311,78 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
 
       if (proc != null) {
         // scalastyle:off println
-        if (proc.isInstanceOf[Driver] || proc.isInstanceOf[SetProcessor] ||
-          proc.isInstanceOf[AddResourceProcessor]) {
-          val driver = new SparkSQLDriver
-
-          driver.init()
-          val out = sessionState.out
-          val err = sessionState.err
-          val start: Long = System.currentTimeMillis()
-          if (sessionState.getIsVerbose) {
-            out.println(cmd)
-          }
-          val rc = driver.run(cmd)
-          val end = System.currentTimeMillis()
-          val timeTaken: Double = (end - start) / 1000.0
-
-          ret = rc.getResponseCode
-          if (ret != 0) {
-            // For analysis exception, only the error is printed out to the console.
-            rc.getException() match {
-              case e : AnalysisException =>
-                err.println(s"""Error in query: ${e.getMessage}""")
-              case _ => err.println(rc.getErrorMessage())
+        proc match {
```

Comment:
to make a smaller pr, can you just use the old style so the indent doesn't need to change?